src/server/queue_func.c

```diff
@@ -112,10 +112,9 @@ que_alloc(char *name)
 	int        i;
 	pbs_queue *pq;
 
-
-	pq = (pbs_queue *)malloc(sizeof(pbs_queue));
+	pq = (pbs_queue *) calloc(1, sizeof(pbs_queue));
 	if (pq == NULL) {
-		log_err(errno, "que_alloc", "no memory");
+		log_err(errno, __func__, "no memory");
 		return NULL;
 	}
 	(void)memset((char *)pq, (int)0, (size_t)sizeof(pbs_queue));
```

Comment:
Can we now remove memset as we are using calloc.

Reply:
reverted to malloc